core/session.ts

```diff
@@ -458,11 +427,14 @@ export class Session {
     }
   }
 
-  public checkTestNameIsUnused(name: string) {
+  private checkTestNameIsUnused(name: string) {
```

Comment:
nit: when i look at these changes in my editor, VSCode gets very unhappy because private members are coming before public - i'm surprised that your VSCode isn't formatting that on save - out of interest, have you ever restarted it?

Reply:
Yes I'm surprised too. Something bad happened recently while I was troubleshooting the linter issue with `includes(...)`. I've had to restart VSCode many times. Sometimes IntelliCode works, sometimes it doesn't...:S

Reply:
We anyway have to change it back to `public` so it can be called from within the actions!